### DISCUSSION TITLE
Disconnect between documentation and code in Renderers

### DISCUSSION THREAD

**Discussion by joshuadavidthomas** [encode/django-rest-framework#8361] [Ideas & Suggestions] [answered]
I've been implementing a custom renderer and I noticed small disconnect in function arguments on the render method between the documentation on Renderers and the actual source code.

In the documentation, the `render` function is shown as taking a `media_type` argument - [docs](https://www.django-rest-framework.org/api-guide/renderers/#media_typenone). However in the `BaseRenderer` in the source code (and every Renderer that inherits from it), the argument is `accepted_media_type` - [source code](https://github.com/encode/django-rest-framework/blob/master/rest_framework/renderers.py#L49).

When I used `media_type`, Pylint screamed at me that the arguments were renamed.

Is it worth it to reconcile the two and change the documentation to match the implentation in the code?

**Comment by lovelydinosaur** [comment, marked as answer]
Heya Josh - good spot!
Yup that looks like it's worth cleaning up.

**Reply by joshuadavidthomas** [reply]
Awesome! I'll open a quick PR if you would like.
